packages/tool_util/setup.cfg

```diff
@@ -55,6 +55,7 @@ console_scripts =
 [options.extras_require]
 cwl =
     cwltool==3.1.20211107152837
+    schema-salad==8.3.20220525163636
```

Comment:
I think I missed why we need to pin schema-salad, isn't it pinned enough in cwltool? Currently the latter has `schema-salad>=8.2.20211104054942,<9` .

Reply:
Anything after this pinned version breaks some tests. I can change it to an upper limit

Reply:
Weird, I can replicate locally the failure of `test_boolean_defaults` unit test with schema-salad==8.3.20220721194857 if I run: `pytest test/unit/tool_util/test_cwl.py` but not if I execute it alone with `pytest test/unit/tool_util/test_cwl.py::test_boolean_defaults`
Not sure if that rings a bell?

I'd prefer not to have to pin schema-salad directly in the long run, if possible.

Reply:
I've done some bisecting and this is the commit that introduces the issue: https://github.com/common-workflow-language/schema_salad/commit/4eda8147e9709a2c0f25da674d4b46f3283f40a6

Any idea @mr-c @jmchilton ?